### PR TITLE
Math magic test

### PR DESCRIPTION
**Pull request by Bushido-brown**
Wrote unite test to test components of the calculator

### CODE REVIEW

File: src/__test__/App.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>,
+    );
+    const calculatorLink = screen.getByText('Calculator');
+    fireEvent.click(calculatorLink);
+    const main = container.querySelector('main');
+    expect(main).toMatchSnapshot();
+  });
+});
```
Comment (roshan-bajgain) [inline src/__test__/App.test.js:43]: - [ ] Your test is not passed as it is throwing an error kindly fix it.
### Error
hiss

File: src/logic/___tests___/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    expect(result).toBe('20');
+  });
+  it('test sum 20 / 5', () => {
+    const result = operate('20', '5', '÷');
+    expect(result).toBe('4');
+  });
+  it('test sum 20 % 5', () => {
+    const result = operate('20', '5', '%');
+    expect(result).toBe('0');
+  });
```
Comment (roshan-bajgain) [inline src/logic/___tests___/calculate.test.js]: - [ ] Kindly write the test for operate.js in a different file for better practice

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Bushido-brown, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Bushido-brown, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters ok
- PR ok
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**